content/docs/enterprise/external-data/zenefits.mdx

```diff
@@ -0,0 +1,118 @@
+---
+title: Zenefits HR systems
+description: Configure the Zenefits external data provider to extend your access policies.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The Zenefits integration provides integration with HR data from Zenefits, such as group memberships, employment status, out of the office, etc.
+
+:::caution
+The external data sources we provide are meant to be examples and inspiration for users to create their own data sources. We won't maintain these integrations in perpituity, and changes to the APIs they interact with may break them in the future.
+
+The [datasource](https://github.com/pomerium/datasource) project is open-source, and if the community wishes to provide contributions to keep it working in the future, we will shepherd those updates.
+:::
+
+## Install
+
+1. Create a Zenefits API key by navigating to **Admin Apps** → **Company Profile** → **Custom Integrations**, and adding a new API Token in the **REST API ACCESS** box. 
+
+  Make sure to only select scopes that you need. As a minimum, `People`, `Status` and `Work email` scopes are required. 
```

Comment:
```suggestion
  Make sure to only select scopes that you need. At a minimum, `People`, `Status` and `Work email` scopes are required. 
```
This might be nitpicky and I'm not even 100% sure I'm correct here.